src/main/java/com/doongji/nestalk/service/user/UserService.java

```diff
@@ -59,4 +59,14 @@ public String findEmailByNameAndPhone(String name, String phone) {
                 .map(User::getEmail)
                 .orElseThrow(() -> new NotFoundException(User.class, name, phone));
     }
+
+    @Transactional
+    public boolean deleteById(Long userId) {
+        User findUser = userRepository.findById(userId).orElseThrow(()-> new NotFoundException("Not Found User"));
+        if (findUser != null){
+            userRepository.deleteById(findUser.getUserId());
```

Comment:
```suggestion
            userRepository.deleteById(userId);
```
컨트롤러에서 Service에 값을 넘길 때, userId값을 넘기고 있으므로
굳이 findById()로 필요하지 않은 사용자 객체를 불러오지 않아도 될것같다는 말이었습니다.

Reply:
아아 그러네요 새롭게 수정했습니다 놓친부분 알려주셔서 감사합니다